### PR TITLE
Issue #58 full screen

### PR DESCRIPTION
**Pull request by fatimzahara**
Add function to put a book when you click to fullscreen mode.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -138,6 +138,11 @@ var library = new Library([BookEPub]);
       //
       console.log("App", "Library", "Opening", entry.title);
       entry.open().then(book => bookViewer.view(book));;
+	  
+	  //
+	  //If the user clicks on a book, it loads in mode fullscreen
```
Comment (Yoric) [inline app.js]: Nit: `// If the user...` (add a whitespace)

File: app.js [hunk 2]
```diff
@@ -138,6 +138,11 @@ var library = new Library([BookEPub]);
       //
       console.log("App", "Library", "Opening", entry.title);
       entry.open().then(book => bookViewer.view(book));;
+	  
+	  //
```
Comment (Yoric) [inline app.js]: Nit: Could you match the indentation of the rest of the file?

Comment (fatimzahara) [inline app.js]: ok I have done to match the indentation.

**Comment by Yoric** [comment]
Could you fold your patches in a single patch?

**Comment by Yoric** [comment]
Looks good, thank you very much.

**Pull request merged**